network/src/network.rs

```diff
@@ -495,42 +486,42 @@ impl NetworkState {
         }
     }
 
-    /// TODO(doc): @driftluo
-    pub fn add_observed_addrs(&self, iter: impl Iterator<Item = Multiaddr>) {
+    /// add observed address for identify protocol
+    pub(crate) fn add_observed_addrs(&self, iter: impl Iterator<Item = Multiaddr>) {
         let mut pending_observed_addrs = self.pending_observed_addrs.write();
         for addr in iter {
             trace!("pending observed addr: {:?}", addr,);
             pending_observed_addrs.insert(addr);
         }
     }
 
-    /// TODO(doc): @driftluo
+    /// network message processing controller, always true, if false, discard any received messages
```

Comment:
Whay `always true`?

Reply:
It is always true unless the user modifies it through rpc

Reply:
Better say "default is true".